ambry-server/src/integration-test/java/com.github.ambry.server/ServerPlaintextTest.java

```diff
@@ -96,9 +96,9 @@ public void endToEndTestWithPrefetch() {
    * Do endToEndTest with backup manager.
    */
   @Test
-  public void endToEndBackupManagerTest() throws Exception {
+  public void endToEndReplicationToCloudBackupTest() throws Exception {
```

Comment:
Replication to CloudBackup? I didn't get the meaning of `Replication To`. Prefer to `EndToEndCloudBackupTest`.